asv_bench/benchmarks/omnisci/benchmarks.py

```diff
@@ -87,10 +131,20 @@ class TimeAppend:
 
     def setup(self, shapes):
         self.df1 = generate_dataframe(
-            ASV_USE_IMPL, "int", *shapes[0], RAND_LOW, RAND_HIGH
+            ASV_USE_IMPL,
+            "int",
+            *shapes[0],
+            RAND_LOW,
+            RAND_HIGH,
+            cache_prefix="left_frame_to_append",
         )
```

Comment:
Can you rewrite dataframes creation like as in `TimeMerge` class(since we will lose the previous results anyway).

Reply:
Done.

BTW, what do you mean under "since we will lose the previous results anyway"

Reply:
After changing the sources, the previous benchmark results are no longer displayed in the dashboard.